data/tlg0732/tlg010/tlg0732.tlg010.1st1K-grc1.xml

```diff
@@ -0,0 +1,5138 @@
+                    <persName>Bruce Robertson</persName>
+                    <resp>Technical Advisor</resp>
+                </respStmt>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Harvard College Library</authority>
+                <idno type="filename">tlg0732.tlg010.1st1K-grc1.xml</idno>
+                <availability>
+                    <p>Available under a Creative Commons Attribution-ShareAlike 4.0 International
+                        License</p>
+                </availability>
+                <date>2017</date>
+                <publisher>Harvard College Library</publisher>
+                <pubPlace>United State</pubPlace>
+            </publicationStmt>
+            <sourceDesc>
+                <listBibl>
+                    <biblStruct>
+                        <monogr>
+                            <title>Supplementum Aristotelicum</title>
+                            <editor>
+                                <persName>
+                                    <name>Ivo Bruns</name>
+                                </persName>
+                            </editor>
+                            <author ref="urn:cts:greekLit:tlg0732">Alexander of Aphrodisias</author>
+                            <imprint>
+                                <publisher>Reimer</publisher>
+                                <pubPlace>Berlin</pubPlace>
+                                <date>1887</date>
+                            </imprint>
+                            <biblScope unit="volume">2</biblScope>
+                        </monogr>
+                        <ref target="http://www.archive.org/stream/p1supplementumar02akaduoft#page/n23/mode/2up">Internet Archive</ref>
+                    </biblStruct>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>Text encoded in accordance with the latest EpiDoc standards (January 2014)</p>
+            <refsDecl n="CTS">
+                <cRefPattern matchPattern="(.+)" n="work"
+                    replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])"
+                />
+            </refsDecl>
+        </encodingDesc>
+        <profileDesc>
+            <langUsage>
+                <language ident="grc">Greek</language>
+                <language ident="lat">Latin</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change/>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="grc" n="urn:cts:greekLit:tlg0732.tlg010.1st1K-grc1">
+                    <div type="textpart" subtype="work" n="1">
+                        <pb n="1"/>
+                        <head>ΑΛΕΞΑΝΔΡΟΥ ΑΦΡΟΔΙΣΙΕΩΣ ΠΕΡΙ ΨΥΧΗΣ.</head>
+                        <note type="marginal">f. 123r Aid.</note>
+                        <p>Ἡ μὲν πρόθεσις ἡμῖν, περὶ ψυχῆς εἰπεῖν τῆς <add cause="fix">τοῦ</add> ἐν
```

Comment:
The work as a whole is too long to have as a single citation unit. I would either add `@n` attributes to the `<p>` elements or enclose them all in `<div>` elements to break the work up and make it more readable online.